<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyFANS</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="plotSplitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="GraphicsLayoutWidget" name="timetracePlot">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="GraphicsLayoutWidget" name="noisePlot"/>
       <widget class="GraphicsLayoutWidget" name="timeNoisePlot"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionWorkingFolder"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionRestoreWindows"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionChannelSettings"/>
    <addaction name="actionOutputSettings"/>
    <addaction name="actionAcquisitionSettings"/>
    <addaction name="actionPowerSupplySettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="controlsWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>195</width>
     <height>91</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Controls</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="1" column="1">
      <widget class="QPushButton" name="stopButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="singleShotButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Single shot</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="startButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="voltagesWidget">
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>400</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Voltages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Drain-Source Voltage</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="SampleStartVoltageUnits">
          <item>
           <property name="text">
            <string>V</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mV</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="SampleStartVoltage"/>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="SampleStepVoltage"/>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="SampleStepVoltageUnits">
          <item>
           <property name="text">
            <string>V</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mV</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Step</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="SampleStopVoltageUnits">
          <item>
           <property name="text">
            <string>V</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mV</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="SampleStopVoltage"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Gate Voltage</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="2">
         <widget class="QComboBox" name="GateStopVoltageUnits">
          <item>
           <property name="text">
            <string>V</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mV</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QComboBox" name="GateStepVoltageUnits">
          <item>
           <property name="text">
            <string>V</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mV</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Step</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="GateStepVoltage"/>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="GateStopVoltage"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="GateStartVoltage"/>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="GateStartVoltageUnits">
          <item>
           <property name="text">
            <string>V</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mV</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Current Values</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Sample Current</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Gate Voltage</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="GateVoltage">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="SampleCurrent">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="SampleVoltage">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Sample Voltage</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Levels</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="GraphicsLayoutWidget" name="histogramPlotLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionWorkingFolder">
   <property name="text">
    <string>WorkingFolder</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>exit.png</normaloff>exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRestoreWindows">
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFrequencySettings">
   <property name="text">
    <string>FrequencySettings</string>
   </property>
  </action>
  <action name="actionAllSettings">
   <property name="text">
    <string>AllSettings</string>
   </property>
  </action>
  <action name="actionFrequency_range">
   <property name="text">
    <string>Frequency range</string>
   </property>
  </action>
  <action name="actionVoltage_range">
   <property name="text">
    <string>Voltage range</string>
   </property>
  </action>
  <action name="actionAll">
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="text">
    <string>SaveAll</string>
   </property>
  </action>
  <action name="actionFrequencies">
   <property name="text">
    <string>Frequencies</string>
   </property>
  </action>
  <action name="actionVoltages">
   <property name="text">
    <string>Voltages</string>
   </property>
  </action>
  <action name="actionSaveAll_2">
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionFrequencies_2">
   <property name="text">
    <string>Frequencies</string>
   </property>
  </action>
  <action name="actionVoltages_2">
   <property name="text">
    <string>Voltages</string>
   </property>
  </action>
  <action name="actionLoadAll">
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionChannelSettings">
   <property name="text">
    <string>Channel settings</string>
   </property>
  </action>
  <action name="actionOutputSettings">
   <property name="text">
    <string>Output settings</string>
   </property>
  </action>
  <action name="actionAcquisitionSettings">
   <property name="text">
    <string>Acquisition settings</string>
   </property>
  </action>
  <action name="actionPowerSupplySettings">
   <property name="text">
    <string>Power Supply Settings</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
